### DISCUSSION TITLE
Private Gpt info

### DISCUSSION THREAD

**Discussion by pedrosl50** [imartinez/privateGPT#1855] [Q&A]
Is it possible with Private GPT to provide reports in Word or PDF written by different individuals discussing the same topic, and ask Private GPT to generate a unified report that combines the two previous reports and is written in a style similar to the PDF reports in the database?

**Comment by BradHutchings** [comment]
It's possible. The question is whether you'll like the results. Here is how I would approach it:
1. Install privateGPT. Go down the path that gets you the mistral-7b-instruct model installed. I have a video that might help you with installation here: https://youtu.be/elBzQAOpQhE
2. Once installed, launch the web interface and add the two files. 
3. Use the "Query Files" mode. Deselect the files to make sure it's using all of them -- I know, that seems backwards.
4. Try prompts to see if it will give you what you need. My first try would be: "Write a report combining the two reports in the context written in the style of the first report."

I hope that's helpful to you.
